zk_toolbox/crates/zk_inception/src/commands/prover/args/init.rs

```diff
@@ -0,0 +1,5 @@
+use clap::Parser;
+use serde::{Deserialize, Serialize};
+
+#[derive(Debug, Clone, Serialize, Deserialize, Parser)]
+pub struct InitArgs {}
```

Comment:
Placeholder.
Should we let the user decide where to store the verification keys?

Reply:
i think we can start without it and introduce it later